zebra-chain/src/block/height.rs

```diff
@@ -144,6 +144,31 @@ impl ZcashDeserialize for Height {
     }
 }
 
+impl ZcashSerialize for Height {
+    fn zcash_serialize<W: io::Write>(&self, mut writer: W) -> Result<(), io::Error> {
+        writer.write_u32::<LittleEndian>(self.0)
+    }
+}
+
+/// A serialized [`Height`].
+pub struct SerializedHeight(Vec<u8>);
+
+impl From<&Height> for SerializedHeight {
+    fn from(height: &Height) -> Self {
+        Self(
+            height
+                .zcash_serialize_to_vec()
+                .expect("The block height should always be serializable."),
+        )
+    }
+}
+
+impl AsRef<[u8]> for SerializedHeight {
+    fn as_ref(&self) -> &[u8] {
+        &self.0
+    }
+}
```

Comment:
Blocker: Making it easier to serialize heights is a good idea. But I think this code is all redundant now. (2/2)